ipatests/test_integration/test_commands.py

```diff
@@ -884,11 +884,10 @@ def test_ssh_from_controller(self):
         if self.master.is_fips_mode:  # pylint: disable=no-member
             pytest.skip("paramiko is not compatible with FIPS mode")
 
-        sssd_version = ''
-        cmd_output = self.master.run_command(['sssd', '--version'])
-        sssd_version = platform_tasks.\
-            parse_ipa_version(cmd_output.stdout_text.strip())
-        if sssd_version.version < '2.2.0':
+        cmd = self.master.run_command(['sssd', '--version'])
+        sssd_version = platform_tasks.parse_ipa_version(
+            cmd.stdout_text.strip())
+        if sssd_version <= platform_tasks.parse_ipa_version('2.2.0'):
```

Comment:
Shouldn't it be like this?
```diff
- if sssd_version <= platform_tasks.parse_ipa_version('2.2.0'):
+ if sssd_version < platform_tasks.parse_ipa_version('2.2.0'):
```
Original comparison is 'if less than'.

Reply:
oh right.. I must've messed it up with this revision by copying from the other place instead of my old 'hacks.diff'